### PR TITLE
split into multiple files; 100% test coverage (closes #75)

### LINKED ISSUE #75: Split into multiple files
- [x] `index.js`
 - [x] `helper.js`
 - [x] ...

### PR DESCRIPTION
**Pull request by omnidan**

### CODE REVIEW

**Comment by pl12133** [comment]
Looks great, 100% coverage is awesome :+1:

**Comment by tiemevanveen** [comment]
Nice, looks great!

File: src/actions.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
+// action types
+export const ActionTypes = {
+  UNDO: '@@redux-undo/UNDO',
+  REDO: '@@redux-undo/REDO',
+  JUMP_TO_FUTURE: '@@redux-undo/JUMP_TO_FUTURE',
+  JUMP_TO_PAST: '@@redux-undo/JUMP_TO_PAST',
+  JUMP: '@@redux-undo/JUMP',
+  CLEAR_HISTORY: '@@redux-undo/CLEAR_HISTORY'
+}
+// /action types
```
Comment (tiemevanveen) [inline src/actions.js]: I think we could drop a lot of these comments now. We all know what it is right? And possible new contributors would either be familiair with redux or should be :p

Comment (omnidan) [inline src/actions.js]: I already dropped some of them in `helper.js`, but you're right, we can drop them here too.

**Pull request merged**